### PR TITLE
Recheck crate names usage

### LINKED ISSUE #731: crate_name vs package_name problem
Currently we have packages on https://crates.io with names like "gtk-sys",
but it include in lib.rs file as `extern crate glib_sys`,
this cause some obscurity for me especially in "codegen/sys/".
I propose fixate these name as next terms:
- `package_name` - name with `-` as in crates.io,
- `crate_name` - name with underscores as in code,

and change Gir codebase accordingly.

IMHO this better done before related issue #685

### PR DESCRIPTION
**Pull request by EPashkin**
Fix #731

No changes in current gtk-rs

cc @GuillaumeGomez, @sdroege

### CODE REVIEW

**Comment by GuillaumeGomez** [comment]
👍

File: src/nameutil.rs [hunk 1]
```diff
@@ -42,6 +42,20 @@ pub fn crate_name(name: &str) -> String {
     }
 }
 
+pub fn exported_crate_name(crate_name: &str) -> String {
```
Comment (sdroege) [inline src/nameutil.rs]: Maybe add some docs to these functions here to explain what they are actually returning and in which context that returned string is used

**Comment by sdroege** [comment]
Looks good to me otherwise

File: src/nameutil.rs (hunk 1 above)
Comment (EPashkin) [reply src/nameutil.rs]: Updated

**Pull request merged**